.github/workflows/ci.yml

```diff
@@ -55,7 +53,16 @@ jobs:
           if grep fedora /etc/os-release ; then
             dnf install -y dotnet-sdk-${{ matrix.dotnet_version }}
           elif grep alpine /etc/os-release; then
+            if grep edge /etc/os-release; then
+              echo "https://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
+            fi
             apk add dotnet-sdk-${{ matrix.dotnet_version }} dotnet-doc
+            if [[ ! ${{ matrix.dotnet_version }} == *6* ]] && [[ ! ${{ matrix.dotnet_version }} == *7* ]]; then
```

Comment:
Today I learned that the `[[ ! $foo == $bar ]]` construct does the same thing as `[[ $foo != $bar ]]`.